Dockerfile

```diff
@@ -1,21 +1,30 @@
 FROM centos:8
+
+
 RUN mkdir /src && \
     yum install -y dnf-plugins-core && \
-    yum config-manager --set-enabled PowerTools && \
+    yum config-manager --set-enabled powertools && \
```

Comment:
This change was needed because the repo name is case sensitive and `PowerTools` was not found